### ISSUE TITLE
Major issues in setup

### ISSUE THREAD

**Issue by doeffgek** [robertklep/homebridge-nefit-easy#47]
Hi Robert,

Ever since the obligation to use child bridges was implemented I have major issues in setting up this plug-in. 

After install I fill in the setup document. For some reason, even in my json it is called nefiteasy it is registered as a neiteasyoutdoortemp.
After a homebridge restart, i can toggle a switch to setup a child bridge, but not after a restart of my homebridge.

Then you can select 'brdige settings', and i get a qr-code to add the bridge. Of course after the device is added to my home i click Save, and yes again we need to restart homebridge.

And then, absolutely not earlier, i have to manually change my json config and delete the 'outdoortemp'. Save the json, and again restart homebridge. 

When I'm lucky this works the first time. I needed 4 attempts today.

Unfortunately I'm getting better at it, because for some reason my homebridge crashes when /var is full. When is clear that issue and give homebridge a manual restart from command line I need to run the entire setup for all my plugins again. For the best part that's not the worst. Yes it's annoying, but your plug-in is a horror to setup. Why is it this complex?

**Comment by robertklep** [comment]
It sounds like it's not my plugin that's a horror to set up, but Homebridge 🤷🏼‍♂️

I haven't used Homebridge for ages, and I also haven't had a Nefit Easy for over 4 years now, so unless someone issues a PR to fix any issues that are caused by newer versions of Homebridge, there's not much I can do about it.

**Comment by doeffgek** [comment]
I agree that homebridge can get on your nerves while setting up. However This plugin is so completely different to set up from any other plugin.I know almost for sure that the problems are from the plugin. When you say it isn’t revised in the last four years how is there a version 2.3.0 and 2.3.1. Tja latter is a recent update so some kind of development and or maintenance is being done. I tried the latest version but it shows slow to no response so I downgraded. Maybe the best solution is a rewrite from scratch. I would be happy to test it since I’ll be needing it for the time being. Grtz GerVerstuurd vanaf mijn iPhoneOp 3 feb 2024 om 19:23 heeft Robert Klep ***@***.***> het volgende geschreven:﻿
It sounds like it's not my plugin that's a horror to set up, but Homebridge 🤷🏼‍♂️
I haven't used Homebridge for ages, and I also haven't had a Nefit Easy for over 4 years now, so unless someone issues a PR to fix any issues that are caused by newer versions of Homebridge, there's not much I can do about it.
